### PR TITLE
cbvabv reduce axiom footprint

### PR DESCRIPTION
**Pull request by wlammen**
1. Remove ax-11 and ax-13 from cbvabv.  This was possible only after a disjoint variables condition was added on x and y.  This had minimal effects, since only one usage did not provide this condition, and this case is now based on cbvab.  On the other side, several theorems have their axiom list reduced.
2. drnfc1 does not need ax-11,  shorten proof by 1 byte.
3. shorten proof of drnfc2 by one byte.
4. nfabd and nfcvf do not need ax-ext, nfabd can also be proven without ax-9.
5. shorten nfabd2.
6. prove cleqf without ax-13.

I'd like to close #2930 since all undisputed shortenings are moved to develop.  Your opinions?

### CODE REVIEW

**Comment by GinoGiotto** [comment]
Is the incorrect counting of steps solved? If not, I would open an issue for it https://github.com/metamath/set.mm/pull/2930#discussion_r1032810265

**Comment by wlammen** [comment]
`Is the incorrect counting of steps solved?`

I did not work on this.

**Comment by icecream17** [comment]
should https://us.metamath.org/mpeuni/cbvabvw.html be removed then?

**Comment by wlammen** [comment]
xx

**Comment by wlammen** [comment]
@icecream17 I think the revision should be attributed to you then.  Unfortunately I have difficulties to update this PR due to some technical problems.  So this have to wait until they are resolved, or the next PR is established in a new fork.

Done
My proof is shorter, so I did not move your version to main.  But I changed the attribution tag.

**Comment by GinoGiotto** [comment]
Regarding the incorrect counting display from metamath.exe I opened an issue [here](https://github.com/metamath/metamath-exe/issues/137)

No objections from me for closing the PR.

File: set.mm
Comment (benjub) [inline set.mm:25935]: Let --> Require.  I would write:
``Require ` x ` and ` y ` to be disjoint to...``
or
``Add a DV condtion on ` x , y ` to...``
(I'm not sure how we write "DV", though: maybe "dv" or maybe "disjoint variable").

File: set.mm
Comment (benjub) [inline set.mm:32755]: It is probably better to add `$d x y $.` here and remove the extra axioms. It looks like the metatheorem that this section on conditional equality aims to prove (more precisely: this section provides the basic theorems that can be used to automate proofs of any instance of that metatheorem) does not require to bundle x and y. Actually, proving all theorems in this section from ax-1--ax-9 would be a nice improvement, if you can.
Do you confirm @digama0 ?

**Comment by wlammen** [comment]
Almost all in this PR is centred around this phrase from [conventions](https://us.metamath.org/mpeuni/conventions.html):
`Minimizing axioms and the axiom of choice. We prefer proofs that depend on fewer and/or weaker axioms, even if the proofs are longer.`
This is more a technical correction than anything else, fully in accordance with our stated goals, and adding nothing that could disturb the development elsewhere.  So I think it can be easily acknowledged.
If you are against this kind of improvements, I suggest to open an issue, so your ideas find an audience and ultimately may lead to a correction of the conventions page.
I still want to know whether #2930 could be closed.

**Comment by benjub** [comment]
> Almost all in this PR is centred around this phrase from [conventions](https://us.metamath.org/mpeuni/conventions.html):
> `Minimizing axioms and the axiom of choice. We prefer proofs that depend on fewer and/or weaker axioms, even if the proofs are longer.`
> This is more a technical correction than anything else, fully in accordance with our stated goals, and adding nothing that could disturb the development elsewhere. So I think it can be easily acknowledged.
> If you are against this kind of improvements, I suggest to open an issue, so your ideas find an audience and ultimately may lead to a correction of the conventions page.
> I still want to know whether #2930 could be closed.

I'm not sure who the "you" above is referring to since no-one submitted a review to this PR yet and it is not apparent who that comment is replying to. My past activity here shows that I am in favor of this kind of improvements, but I have not been reviewing PRs for a while, and I don't intend to do it again soon. In fact, just in case the "you" was referring to me, I want to be clear:

I have not submitted any review to any PR, that is, I have not approved nor rejected any PR, since, I think, the beginning of March, more than two months ago (apart from maybe occasional work on the LaTeX part because my help was needed, and a few things that were awaiting for action to unblock stuff). I have only made a few comments, in the hope that this could help improve database quality here and there, and I wrote some Issues to try and give some advice (often asking also the point of view of @digama0, who is the other person well-versed in math here). But apparently, even that is too much, as I seem to understand from the tone of some earlier comments made by @wlammen or @jkingdon. Therefore, I will spare me some time and focus on other stuff where I have more added value (but I will not completely forget what's in my mathboxes, nor Metamath). Incidentally, the reason I stopped reviews, or the straw that broke the camel's back, was that answer from @david-a-wheeler https://github.com/metamath/set.mm/issues/3054#issuecomment-1462932722 which was at the same time funny but also discouraging. So to answer your above "your ideas find an audience", I'm afraid this is whishful thinking.

**Comment by wlammen** [comment]
I did not refer to you personally, @benjub , but more to the fact that this PR is without approval/disapproval now for about 3 days, and I was simply guessing about the reason.

`But apparently, even that is too much, as I seem to understand from the tone of some earlier comments made by wlammen ... `.  I am not sure how to react on this.  First, I'd like to know which of my comments made you feel like this, just for the purpose of learning, and improving my style of language.  If you don't want to display them in the open here, you may contact me at Wolf dot Lammen at gmail dot com.

Second, I'd like to find a strategy to keep people active in this project.  Suggestions/opinions are welcome in #3170 .

**Review by jkingdon — APPROVED** [review]

**Comment by jkingdon** [comment]
Looks to me like all the comments on this pull request have been addressed or are non-blocking in terms of merging what is here.

Do people agree we can merge this, whatever else should happen in the future?

**Comment by wlammen** [comment]
Thanks @jkingdon .  I will look into #3167 tomorrow.  Its too big for me to do in one sweep.

File: set.mm
Comment (digama0) [reply set.mm:32755]: I would be okay with either approach here. On the one hand, it is nice to have more general theorems, but on the other hand it is also nice to use fewer axioms, and I think you are correct to assert that the proof does not need these lemmas to lack a `$d x y` in them.

**Pull request merged**
